app_test.go

```diff
@@ -546,8 +546,7 @@ func TestApp_CommandWithFlagBeforeTerminator(t *testing.T) {
 
 	expect(t, parsedOption, "my-option")
 	expect(t, args[0], "my-arg")
-	expect(t, args[1], "--")
```

Comment:
Could use extra eyes on this change; it's a result of the latest commit / fix, but could potentially be a breaking change for users (so _at least_ should be called out in the release notes); I _think_ I interpreted the POSIX spec correctly with my patch, but specs can sometimes be ambiguous, so could use the extra eyes.

I also haven't checked the `v2` branch/version yet (possibly the fix is needed there as well, or in another form)

Reply:
So far, the twitter-verse seems to consolidate to my interpretation being correct 😂  https://twitter.com/thaJeztah/status/1260921988102656002?s=20

Reply:
I'm having a hard time imagining is this is a mundane change or if it'll cause mass chaos for somebody somewhere. Is there a way to keep this behavior as-is???